src/components/application_manager/src/commands/mobile/get_vehicle_data_request.cc

```diff
@@ -268,28 +268,46 @@ void GetVehicleDataRequest::on_event(const event_engine::Event& event) {
   switch (event.id()) {
     case hmi_apis::FunctionID::VehicleInfo_GetVehicleData: {
       EndAwaitForInterface(HmiInterfaces::HMI_INTERFACE_VehicleInfo);
+
+      smart_objects::SmartObject incoming_msg_params = message[strings::msg_params];
+      
+      if (incoming_msg_params.keyExists(strings::gps)) {
+          if (incoming_msg_params[strings::gps].empty()) {
+              LOG4CXX_ERROR(logger_, "Invalid response received from system");
+
+              SendResponse(false,
```

Comment:
@KVGrygoriev according to our code style https://google.github.io/styleguide/cppguide.html#Naming
would be nice to add comment to magic value
SendResponse(/*var_name=*/false, ...